src/mapInfoContributorManager.cpp

```diff
@@ -153,39 +155,41 @@ MapInfoProperties MapInfoContributorManager::fullInfo(int roomID, int selectionS
+             * rooms are selected (and the focus is on the current "player room"): %1 is the
+             * room ID number, and %2, %3, %4 are the x, y, and z coordinates for that room.
+             */
+            infoText.append(tr("Room\u00A0ID:\u00A0%1 Position\u00A0on\u00A0Map: (%2,%3,%4) \u2011\u00A0current player location")
                                     .arg(QString::number(roomID),
-                                        QString::number(room->x()),
-                                        QString::number(room->y()),
-                                        QString::number(room->z()))));
+                                         QString::number(room->x()),
+                                         QString::number(room->y()),
+                                         QString::number(room->z()))
+                                    .append(QChar::LineFeed));
             if (areaId != displayAreaId) {
                 isItalic = true;
             } else {
                 isBold = true;
             }
             break;
         case 1:
-            infoText.append(qsl("%1\n").arg(
-                /*:
-                This text uses non-breaking spaces (Unicode U+00A0) and a non-breaking hyphen which
-                are used to prevent the line being split at some places it might otherwise be.
-                When translating, please consider at which points the text may be divided to fit onto
-                more than one line. This text is for when ONE room is selected: %1 is the room ID number,
-                and %2, %3, %4 are the x, y, and z coordinates for the selected room.
-                */
-                               tr("Room\u00A0ID:\u00A0%1 Position\u00A0on\u00A0Map: (%2,%3,%4) ‑\u00A0selected room")
+            /*:
+             * This text uses non-breaking spaces (Unicode U+00A0) and a non-breaking hyphen
+             * (Unicode U+2011) which are used to prevent the line being split at some places
+             * it might otherwise be. When translating, please consider at which points the
+             * text may be divided to fit onto more than one line. This text is for when ONE
+             * room is selected: %1 is the room ID number, and %2, %3, %4 are the x, y, and z
+             * coordinates for that selected room.
+             */
+            infoText.append(tr("Room\u00A0ID:\u00A0%1 Position\u00A0on\u00A0Map: (%2,%3,%4) \u2011\u00A0selected room")
                                     .arg(QString::number(roomID),
-                                        QString::number(room->x()),
-                                        QString::number(room->y()),
-                                        QString::number(room->z()))));
+                                         QString::number(room->x()),
+                                         QString::number(room->y()),
+                                         QString::number(room->z()))
+                                    .append(QChar::LineFeed));
```

Comment:
Nice hack

Reply:
Yeah, there are other places where a raw line-feed (or even a `QLatin1String("\n")`) could be replaced with this.